handsontable/src/plugins/nestedHeaders/__tests__/nestedHeaders.spec.js

```diff
@@ -387,411 +387,6 @@ describe('NestedHeaders', () => {
       expect(levels[3][2]).toEqual(getCell(-1, 2));
       expect(levels[3][3]).toEqual(getCell(-1, 3));
     });
-
-    it('should render the setup properly after the table being scrolled', () => {
```

Comment:
Here, I removed the test as it duplicates that one https://github.com/handsontable/handsontable/pull/9559/files#diff-ca95e8f246d20763bb146012951991bed5cfff4a011e94fe3fa978f65299f70eR520.